sale_timesheet_invoice_description/models/sale.py

```diff
@@ -29,7 +29,12 @@ class SaleOrderLine(models.Model):
     def _prepare_invoice_line_details(self, line, desc_rule):
         details = []
         if desc_rule[0] == '1':
-            details.append(line.date)
+            # If project_timesheet_time_control module is installed,
+            # date field is hidden and the new date_time field is used instead
+            if hasattr(line, 'date_time'): # pragma: no cover
```

Comment:
Although the field is hidden, `date` field is still synchronized, so I prefer to keep the previous behavior showing only date. Customer doesn't need this kind of detail.

Reply:
Yes, I agree, thanks